flow/project.py

```diff
@@ -532,11 +653,19 @@ def __init__(self, cmd, pre=None, post=None):
     def __str__(self):
         return "{type}(cmd='{cmd}')".format(type=type(self).__name__, cmd=self._cmd)
 
-    def __call__(self, job=None):
+    def __call__(self, *jobs):
+        if len(jobs) == 0:
```

Comment:
I think you want something like this:
```suggestion
    def __call__(self, *jobs, **kwargs):
        job = kwargs.pop('job', None)
        if kwargs:
            raise ValueError(f"Invalid key-word arguments: {', '.join(kwargs)}")
        if job is not None:
            warnings.warn(...)
```